.editorconfig

```diff
@@ -9,6 +9,15 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+# https://github.com/CppCXY/EmmyLuaCodeStyle/blob/master/docs/format_config_EN.md
+[*.lua]
+quote_style = double
+call_arg_parentheses = remove_table_only
+max_line_length = 180
```

Comment:
Nope nope nope.

Reply:
PS Sorry, forgot to say. But no dumb algorithm shall impose itself on our precious humans. ;-)

Reply:
OK, sorry, I thought this would help those new to the KOReader code base (like myself) follow along the style of existing code more easily. But never mind, I'll close this

Reply:
People may split lines to improve legibility. Algorithms that split lines on the basis of meaningless attributes like line length do no such thing by definition, and they are therefore undesirable to the highest degree. They actively harm maintainability.